frontend/src/APIClients/mutations/UserMutations.ts

```diff
@@ -0,0 +1,16 @@
+import { gql } from "@apollo/client";
+
+const UPDATE_USER = gql`
+  mutation UpdateUser($id: ID!, $user: UpdateUserDTO!) {
+    updateUser(id: $id, user: $user) {
+      id
+      firstName
+      lastName
+      email
+      town
+      role
+    }
```

Comment:
nit: this is a lot of fields to pass to the backend. It might be better to make a specific `UpdateUserRole` mutation for this purpose.

Reply:
Yeah, could be good to scope it here from the frontend, but I find it still ends up being somewhat redundant for any other updates.